site/en/blog/spc-on-android/index.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Secure Payment Confirmation on Chrome Android
+subtitle: Secure Payment Confirmation on Chrome Android is shipping in Chrome 109.
+description: >
+  Secure Payment Confirmation brings a phishing resistant payment confirmation with the power of passkeys to the web. It will be available in Chrome Android from Chrome 109.
+hero: image/YLflGBAPWecgtKJLqCJHSzHqe2J2/0uN2JSfRTgF90W89BzMX.jpg
+alt: Phone/Lock/Safe.
+authors:
+  - smcgruer
+date: 2022-12-01
+---
+
+## Overview
+
+Secure Payment Confirmation (SPC) is a [proposed web
+standard](https://www.w3.org/TR/secure-payment-confirmation/) that allows
+customers to authenticate with a credit card issuer, bank, or other payment
+service provider using a platform authenticator—typically activated with a
+device's screen unlock feature such as a fingerprint sensor. This usually
+happens during a payments authentication protocol such as [EMV 3-D
+Secure](https://www.emvco.com/emv-technologies/3d-secure/) or [Open
+Banking](https://standards.openbanking.org.uk/). EMV 3-D Secure, for example,
+has support for SPC in its [v2.3 spec
+release](https://www.emvco.com/emv_insights_post/what-is-new-with-emv-3ds-v2-3/).
+We [previously announced](/articles/secure-payment-confirmation/) that SPC was
+launched for Google Chrome on macOS and Windows and provided developer guides
+for both [registration](/articles/register-secure-payment-confirmation/) and
+[authentication](/articles/authenticate-secure-payment-confirmation/).
+
+As of M109 (currently on the Beta channel), SPC will also be available on Google 
+Chrome on Android. Users will be able to use their device's screen lock to 
+complete the payment verification process on merchant sites that use SPC.
+
+<figure class="screenshot" style="max-width:300px; margin:auto;">
+  {%  
+    Video src="video/YLflGBAPWecgtKJLqCJHSzHqe2J2/M1yhlVlS27eBDM25XYSt.mp4",  
+    autoplay="true", loop="true"  
+  %}
+  <figcaption>A user is paying on Chrome using the Secure Payment Confirmation.</figcaption>
+ </figure>
+
+If you are interested in experimenting with SPC, feel free to try it out on [our
+demo website](https://spc-merchant.glitch.me/), or ask your Payment Service
+Provider if they plan to support it for authenticating user payments.
+
+{% Aside %}
+
+As SPC is specialised in payment authentication, you can use this API directly
+only if you are a credit card issuer, bank, or other payment service provider.
+Otherwise, please reach out to your credit card issuer, bank or payment service
+provider if you are interested in using this feature.
+
+{% endAside %}
+
+## Strong Authentication for payments
```

Comment:
Just one thing to double-check—are we using "Strong Authentication" as an official name—is that why we have it in title case?

I'm aware of [SCA](https://www.fca.org.uk/firms/strong-customer-authentication), is that the reference being made here? If not, should it not be "Strong authentication for payments"?

Reply:
Chatted offline. We'll use "Strong **a**uthentication" to avoid conflicting with the SCA.